lua/refactoring/treesitter/langs/lua.lua

```diff
@@ -17,9 +17,11 @@ function Lua.new(bufnr, ft)
             local_function = "function",
             ["function_declaration"] = "function",
             function_definition = "function",
+            chunk = "chunk",
```

Comment:
Are the changes in this file still required? I don't think I see them referenced anywhere else in the PR

Reply:
Yes, they are needed. The global_scope test case that I added for Lua fails because scope is nil without it.